### ISSUE TITLE
Region Feature Matrix Extraction as a Pre-Transform

### ISSUE THREAD

**Issue by aligholami** [pyg-team/pytorch_geometric#920]
Hi, I wonder if there is any dataset pre-transforms to perform region feature matrix extraction and set them as nodes of the graph?

Thanks!

**Comment by rusty1s** [comment]
No, there is no such functionality, you need to provide your own. I would welcome a PR though :)
